docs/uploader/xhr.mdx

```diff
@@ -87,7 +87,14 @@ URL of the HTTP server (`string`, default: `null`).
 #### `method`
 
 Configures which HTTP method to use for the upload (`string`, default:
-`'post'`).
+`'POST'`).
+
+:::note
+
+Using non-uppercase string for `method` is deprecated and won’t be supported in
+future versions of Uppy.
+
+:::
```

Comment:
Costs us nothing to maintain the `toUpperCase` there so might as well leave it 🤷 
```suggestion
```

Reply:
I'm not sure I agree, it forces us to create a new object just to override the user provided value, otherwise we can just pass it along and shave a few bytes from the bundle size.

Reply:
In the big bundle size that is Uppy, not creating a new single object is an imaginary performance improvement. If it costs us practically speaking nothing and it isn't a problem, it's better to not introduce a breaking change and leave it as is.

Reply:
Actually after reading the spec, it seems there are no reason the upper case the methods. Standards methods are normalized (i.e. upper-cased) as spec'd in https://fetch.spec.whatwg.org/#concept-method. Uppy was wrong to call `.toUpperCase()`. I think we can get away with only documenting that Uppy is not following the standard on the 3.x branch

Reply:
Not sure if the fetch spec applies to `XMLHttpRequest`?

Reply:
See https://xhr.spec.whatwg.org/#the-open()-method, step 4 is linking to the fetch spec.

Reply:
```suggestion
:::note

Uppy is not following the standard and will uppercase the method name. This behavior will be removed in a future version to align Uppy with the web APIs.

:::
```